validatorclient/src/main/java/tech/pegasys/artemis/validatorclient/ValidatorClient.java

```diff
@@ -13,7 +13,68 @@
 
 package tech.pegasys.artemis.validatorclient;
 
+import static tech.pegasys.artemis.datastructures.util.BeaconStateUtil.get_crosslink_committees_at_slot;
+import static tech.pegasys.artemis.datastructures.util.BeaconStateUtil.get_current_epoch;
+import static tech.pegasys.artemis.datastructures.util.BeaconStateUtil.get_epoch_start_slot;
+import static tech.pegasys.artemis.datastructures.util.BeaconStateUtil.get_previous_epoch;
+
+import com.google.common.primitives.UnsignedLong;
+import java.util.ArrayList;
+import java.util.List;
+import tech.pegasys.artemis.datastructures.state.BeaconState;
+import tech.pegasys.artemis.datastructures.state.CrosslinkCommittee;
+
 public class ValidatorClient {
 
+  private static final int SLOTS_PER_EPOCH = 64; // 2^6
+
   public ValidatorClient() {}
+
+  /**
+   * Return the committee assignment in the ``epoch`` for ``validator_index`` and
+   * ``registry_change``. ``assignment`` returned is a tuple of the following form: *
+   * ``assignment[0]`` is the list of validators in the committee * ``assignment[1]`` is the shard
+   * to which the committee is assigned * ``assignment[2]`` is the slot at which the committee is
+   * assigned * ``assignment[3]`` is a bool signalling if the validator is expected to propose a
+   * beacon block at the assigned slot.
+   *
+   * @param state the BeaconState.
+   * @param epoch either on or between previous or current epoch.
+   * @param validator_index the validator that is calling this function.
+   * @param registry_change whether there has been a validator registry change.
+   * @return
+   */
+  public CommitteeAssignmentTuple get_committee_assignment(
+      BeaconState state, int epoch, int validator_index, boolean registry_change) {
+    int previous_epoch = get_previous_epoch(state).intValue();
+    int next_epoch = get_current_epoch(state).intValue();
```

Comment:
It's not the best idea to use int values. It would be great honestly if you could compare them as unsigned longs.